docs/guides/storage_clients.mdx

```diff
@@ -14,6 +14,8 @@ import FileSystemStorageClientBasicExample from '!!raw-loader!roa-loader!./code_
 import FileSystemStorageClientConfigurationExample from '!!raw-loader!roa-loader!./code_examples/storage_clients/file_system_storage_client_configuration_example.py';
```

Comment:
Add the new `SqlStorageClient` to the bullet list in the "Built-in storage clients" section please.